src/metabolic_ninja/tasks.py

```diff
@@ -117,13 +122,31 @@ def predict(self, model_obj, product_name, max_predictions, aerobic,
     #  default_biomass_reaction. Maybe we need a new field for the medium
     #  database model?
     model.carbon_source = "EX_glc__D_e"
+    # Retrieve user details. This should be retrieved here, to ensure that the
```

Comment:
In terms of code organization, it seems to me like this new section should go into the `notify` task.

Reply:
The reason it's not is that the forwarded JWT required to retrieve user details may have expired by the time the `notify` task executes